### PR TITLE
MK3: Update calibration status v2 documentation

### PR DESCRIPTION
**Pull request by 3d-gussner**
image

- [x] Cherry-pick to MK3_3.12 after merge of https://github.com/prusa3d/Prusa-Firmware/pull/3861

### CODE REVIEW

**Review by DRracer — APPROVED** [review]

File: Firmware/eeprom.h [hunk 1]
```diff
@@ -349,6 +349,16 @@ static_assert(sizeof(Sheets) == EEPROM_SHEETS_SIZEOF, "Sizeof(Sheets) is not EEP
...
+| ^           | ^       | ^                                     | 08h 8        | ^                     | Temp model cal passed                             | ^            | ^
+| ^           | ^       | ^                                     | 10h 16       | ^                     | Live Adjust set                                   | ^            | ^
+| ^           | ^       | ^                                     | 20h 32       | ^                     | Free bit                                          | ^            | ^
+| ^           | ^       | ^                                     | 40h 64       | ^                     | Free bit                                          | ^            | ^
+| ^           | ^       | ^                                     | 80h 128      | ^                     | Free bit                                          | ^            | ^
+| ^           | ^       | ^                                     | 1fh 31       | ^                     | All Wizard steps passed                           | ^            | ^
```
Comment (wavexx) [inline Firmware/eeprom.h]: I would remove 0. 1fh is not possible. 80h is "unknown".
We effectively have two bits left.

**Review by wavexx — CHANGES_REQUESTED** [review]

File: Firmware/eeprom.h (hunk 1 above)
Comment (3d-gussner) [reply Firmware/eeprom.h]: 1fh = 31 it is the value after you finish the wizard see https://www.rapidtables.com/convert/number/hex-to-decimal.html?x=1f

Comment (wavexx) [reply Firmware/eeprom.h]: Oh, in that sense. Mmh, I would only list bit values here, like the values before. "Fully calibrated" will change over time.

**Comment by wavexx** [comment]
Can you rebase on MK3?

**Comment by DRracer** [comment]
rebased onto MK3

**Review by wavexx — APPROVED** [review]

**Pull request merged**